hapi/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ plugins {
     id("com.hedera.gradle.protobuf")
     id("com.hedera.gradle.services-publish")
     id("com.hedera.gradle.feature.test-fixtures")
-    alias(libs.plugins.pbj)
+    id("com.hedera.pbj.pbj-compiler") version "0.9.2"
```

Comment:
Because this is a plugin version (which Gradle needs early during configuration). This cannot go into `hedera-dependency-versions`. Since this is something **only** used in the `hapi` project, I think it is best to define the version here.